app/action/real-time-client-action.coffee

```diff
@@ -6,7 +6,8 @@ xhrPromise = require '../util/xhr-promise'
 getTopic = (options) ->
   route = if options.route then options.route else "+"
   direction = if options.direction then parseInt(options.direction) + 1  else "+"
-  '/hfp/journey/+/+/' + route + '/' + direction + '/#'
+  trip = if options.trip then options.trip else "+"
```

Comment:
I have wondered this row before. Why options might or might not contain ".trip", I probably know the answer, but we should comment this